lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -308,7 +309,7 @@ def _on_scrape_targets_changed(self, event):
 `scrape_jobs` and `alert_rules` keys in application relation data
 of Metrics provider charms hold eponymous information.
 
-"""
+"""  # noqa: W505
```

Comment:
I'm approving but you should fix the doc string length or we should remove the requirement.